code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -371,6 +371,21 @@ Defined in conflicts.dm of the #defines folder.
 	accuracy_mod = CONFIG_GET(number/combat_define/min_hit_accuracy_mult)
 	scatter_mod = -CONFIG_GET(number/combat_define/low_scatter_value)
 
+/obj/item/attachable/mosinbarrel
+	name = "sniper barrel"
+	icon_state = "mosinbarrel"
+	desc = "A heavy barrel. CANNOT BE REMOVED."
+	slot = "muzzle"
+
+	pixel_shift_x = 20
+	pixel_shift_y = 16
+	flags_attach_features = NOFLAGS
+
+/obj/item/attachable/mosinbarrel/Initialize()
+	. = ..()
+	accuracy_mod = CONFIG_GET(number/combat_define/min_hit_accuracy_mult)
```

Comment:
this should be in `set_gun_config_values()`

Reply:
the four attachment definitions are not available in shotguns.dm. However I was able to remove the duplicate definition on the gun_attachable.dm side.